kubectl-plugin/README.md

```diff
@@ -18,22 +18,79 @@ The general command structure is `kubectl mayastor <operation> <resource>` where
 
 The plugin needs to be able to connect to the REST server in order to make the appropriate REST calls. The IP address and port number of the REST server can be provided through the use of the `--rest` command line argument. If the `--rest` argument is omitted, the plugin will attempt to make use of the kubeconfig file to determine the IP of the master node of the cluster. Should the kubeconfig file contain multiple clusters, then the first cluster will be selected.
 
-### Examples
 
-#### Volumes
-Getting all volumes:\
-`kubectl mayastor get volumes`
+### Examples and Outputs
 
-Getting a specific volume:\
-`kubectl mayastor get volume ec4e66fd-3b33-4439-b504-d49aba53da26`
+1. List Volumes
```

Comment:
I think we want to change `List` to `Get` because the operation is now a `Get` operation.

Reply:
From the word `List` here I meant the operation and not the command, I have mentioned the command along with the output. I will change it to Get to avoid confusions.